### PR TITLE
fix: `hackernews_js_fetch` example for `leptos_0.7`

### PR DESCRIPTION
**Pull request by sabify**

### CODE REVIEW

File: examples/hackernews_js_fetch/src/lib.rs [hunk 1]
```diff
@@ -1,40 +1,67 @@
...
     provide_meta_context();
+    let (is_routing, set_is_routing) = signal(false);
+
     view! {
-        <Link rel="shortcut icon" type_="image/ico" href="/public/favicon.ico"/>
-        <Stylesheet id="leptos" href="/public/style.css"/>
-        <Meta name="description" content="Leptos implementation of a HackerNews demo."/>
-        <Router>
-            <Nav />
+        <head>
```
Comment (gbj) [inline examples/hackernews_js_fetch/src/lib.rs:36]: This extra `<head>` tag should be removed -- you create the `<head>` in the `shell` above.

**Review by gbj — CHANGES_REQUESTED** [review]
Thanks. If you run it you'll notice the JS/WASM don't load. This is because `.leptos_routes()` needs to be updated so that it serves the `shell()`, not just the body.

```rust
                .leptos_routes(&leptos_options, routes, {
                    let options = leptos_options.clone();
                    move || shell(options.clone())
                })
```

See also comment re duplicated `<head>`

File: examples/hackernews_js_fetch/src/lib.rs (hunk 1 above)
Comment (sabify) [reply examples/hackernews_js_fetch/src/lib.rs]: Could you please confirm that it is working now?

**Comment by gbj** [comment]
Thanks! This is great. Much appreciated.

**Pull request merged**